.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,7 +26,7 @@ If applicable, add screenshots to help explain your problem.
 **Desktop (please complete the following information):**
  - OS: [e.g. Mac, Window, Linux]
  - Browser [e.g. chrome, firefox]
- - Node Version [e.g. 18.12.0]
+ - Node Version [e.g. 18.19.1]
```

Comment:
👍🏼 

We need to update in `package.json`

<img width="309" alt="스크린샷 2024-08-28 오후 5 35 58" src="https://github.com/user-attachments/assets/d3b5d65b-fb62-4274-abf0-a84aa1e78c5e">

Reply:
Also here:

![image](https://github.com/user-attachments/assets/56094111-a3af-41f6-98b6-64df24f9878e)

Anyway we need to bump it to 20+ (22 isn't working 100% correctly for now, we can wait a while and bump it directly to 22+)